python/eos/tasks.py

```diff
@@ -15,6 +15,8 @@
 # this program; if not, write to the Free Software Foundation, Inc., 59 Temple
 # Place, Suite 330, Boston, MA  02111-1307  USA
 
+from __future__ import annotations
```

Comment:
Can you add a comment why you need this for old version (so that we remove it at some point)?

Reply:
Yes

Reply:
Removed, due to problems with the annotations. Returning to plain ``list`` hints, rather than ``list[float]`` or similar.